functions-and-scope.js

```diff
@@ -39,15 +67,43 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Wat moet ik verzamelen uit de array van cijfers om uiteindelijk een gemiddelde te kunnen berekenen?
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
+console.log("----2a")
+let totalNumber = 0
+
+for (let i = 0; i < grades.length; i++) {
+    totalNumber = totalNumber + grades[i]
```

Comment:
Dit kun je wat korter schrijven
```suggestion
    totalNumber += grades[i];
```